cvae/README.md

```diff
@@ -0,0 +1,157 @@
+# Convolutional Variational Autoencoder (CVAE) on MNIST
+
+## About
+
+Convolutional variational autoencoder (CVAE) implementation in MLX using MNIST. This is a minimal example ported from the [torch-vae](https://github.com/menzHSE/torch-vae) repo. 
```

Comment:
I would prefer not to cite personal repos in MLX examples. Is that ok with you? Would rather keep acknowledgements to the ACKNOWLEDGMENTS.md file?